package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {    
+      "react/react-in-jsx-scope": "off",
+      "semi" : ["warn", "always", { "omitLastInOneLineBlock" : true }]
+    },
+    "globals": {    
+      "React": "writable"
```

Comment:
Found here https://medium.com/@joelmasters/setting-up-eslint-for-nextjs-37163d4cabaa

Reply:
Removed as per @psarando's suggestion.